lib/main.dart

```diff
@@ -20,3 +79,13 @@ void main() async {
     todosRepository: todosRepository,
   ));
 }
+
+Future<void> _firebaseMessagingBackgroundHandler(RemoteMessage message) async {
+  await Firebase.initializeApp();
+  log("Handle messages ");

```

Comment:
should we not be showing a notification screen?

Reply:
Foreground messages are being handled in the notifier component, when in the foreground this receives messages when the app is not running or is in the background.

Reply:
Added the ui in the notifier